### PR TITLE
Amend card structure for Features section on Homepage

### PR DESCRIPTION
**Pull request by abhigyadufare**
Card structure for Features section on Homepage.

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
Hi @abhigyadufare, seems like Netlify Build time is over , you can regenerate the build process with - 

```bash
git reset
git commit --amend --no-edit
git push --force-with-lease
```

**Comment by abhigyadufare** [comment]
Hi @utkarsh-singh1, i followed your instructions and force-pushed it.

**Comment by sftim** [comment]
/retitle Amend card structure for Features section on Homepage

**Comment by sftim** [comment]
This may relate to #38958

**Comment by sftim** [comment]
@abhigyadufare please write a PR description that summarizes why we should make this change

**Comment by abhigyadufare** [comment]
@sftim okay

**Comment by abhigyadufare** [comment]
### As mentioned in the description of the issue #38958 

**This is a Feature Request** which has been approved

212727755-884a255e-fa84-46e2-bb5c-a455f1712d08

The above image displays the card structure that needs to be implemented

**Why is this needed**
It provides a better structure and look to the section. It makes them stand out.

File: package-lock.json
Comment (sftim) [inline package-lock.json:1]: Please don't delete this file.

**Review by sftim — CHANGES_REQUESTED** [review]
Looks good, _but_ I don't want to delete the JavaScript package lock.

File: package-lock.json
Comment (abhigyadufare) [reply package-lock.json:1]: i used 
-`npm audit fix`
but i can remove it if required and keep the previous package.json file.

**Comment by abhigyadufare** [comment]
@sftim Is this what you wanted me to do?

**Comment by sftim** [comment]
Thanks

/lgtm

**Review by divya-mohan0209 — COMMENTED** [review]
LGTM! Thank you so much for these changes :)
/approve

**Comment by abhigyadufare** [comment]
> LGTM! Thank you so much for these changes :) /approve

@divya-mohan0209  Pleasure 😊 Please do me know if i should do this on other pages as well.

**Comment by sftim** [comment]
Feel free to open new issues with suggested web design improvements @abhigyadufare

**Pull request merged**